src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck ford = new Truck();
```

Comment:
```suggestion
        Machine ford = new Truck();
```
для того щоб використовувати поліморфізм потрібно оголошувати посилання спільного більш абстрактного типу ( інтерфейс абстрактний клас ...)